src/App.jsx

```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import PuzzlePage from "./containers/PuzzlePage/index";
 import { useTranslation } from "react-i18next";
+import PuzzlePage from "./containers/PuzzlePage/index";
```

Comment:
app.jsx shouldnt be changed in a PR for the success card

Reply:
renamed PR